DariaGuketleva/blog/src/app/pages/Comment.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import CommentContent from '../components/Post';
```

Comment:
```suggestion
import CommentContent from '../components/Comment';
```

нельзя будет отдельно открыть комментарий